### PR TITLE
Add PoolPositionModal on Portfolio page

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v3/src/components/App/App.tsx [hunk 1]
```diff
@@ -51,6 +51,10 @@ const App = () => {
           <div className="tc__app__body">
             <Routes>
               <Route path="/portfolio" element={<Portfolio />} />
+              <Route
+                path="/position/details/:chain/:ticker/:protocol/:poolAddress"
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/components/App/App.tsx:55]: Any reason to use `/position/details`? Could be only `position`?

**Review by ShawTim — APPROVED** [review]

File: packages/tempus-client_v3/src/components/App/App.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/components/App/App.tsx:55]: Yes, it can be position only. I will change it.

Comment (lazarmitic) [reply packages/tempus-client_v3/src/components/App/App.tsx]: fixed

**Review by danieledamiani — APPROVED** [review]

**Review by ShawTim — APPROVED** [review]

File: packages/tempus-client_v3/src/components/Portfolio/PortfolioPositions/PortfolioPositions.tsx [hunk 2]
```diff
@@ -40,7 +43,18 @@ const PortfolioPositions: FC = () => {
     return groups;
   }, [balances, chain, tempusPools]);
 
-  const handleCardClick = useCallback(() => {}, []);
+  const handleCardClick = useCallback(
+    (cardChain: Chain, ticker: Ticker, protocol: ProtocolName, _status: PoolCardStatus, poolAddresses: string[]) => {
+      if (poolAddresses.length === 1) {
+        const poolAddress = poolAddresses[0];
+
+        navigate(`/position/${cardChain}/${ticker}/${protocol}/${poolAddress}`);
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/Portfolio/PortfolioPositions/PortfolioPositions.tsx:51]: Should it be `/portfolio/position/${cardChain}/${ticker}/${protocol}/${poolAddress}` now?

Comment (lazarmitic) [reply packages/tempus-client_v3/src/components/Portfolio/PortfolioPositions/PortfolioPositions.tsx]: Yup :D

**Review by iamvukasin — APPROVED** [review]

**Pull request merged**